### PR TITLE
Task #165159 feat: Build user plugin for TJNotification. On profile u…

### PR DESCRIPTION
**Pull request by shindebalu**
…pdate/user creation, the plugin will add an SMS subscription in TJNotification

### CODE REVIEW

File: src/plugins/user/tjnotificationsmssub/tjnotificationsmssub.php [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+use Joomla\CMS\MVC\Model\BaseDatabaseModel;
+
+BaseDatabaseModel::addIncludePath(JPATH_ADMINISTRATOR . '/components/com_tjnotifications/models', 'SubscriptionsModel');
+
+/**
+ * Class for Tjnotificationsmssub User Plugin
+ *
+ * @since  1.0.0
+ */
+class PlgUserTjnotificationsmssub extends CMSPlugin
```
Comment (manojLondhe) [inline src/plugins/user/tjnotificationsmssub/tjnotificationsmssub.php:28]: rename plugin as tjnotifications_mobilenumber

File: src/plugins/user/tjnotificationsmobilenumber/tjnotificationsmobilenumber.php [hunk 2]
```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * @package     Joomla.Plugin
```
Comment (manojLondhe) [inline src/plugins/user/tjnotificationsmobilenumber/tjnotificationsmobilenumber.php:3]: Change pkg as
 * @package     Tjnotifications

File: src/plugins/user/tjnotificationsmobilenumber/tjnotificationsmobilenumber.php [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+	 */
+	public function onUserAfterSave($user, $isNew, $success, $msg)
+	{
+		$notificationsParams = ComponentHelper::getParams('com_tjnotifications');
+		$phoneSetting        = $notificationsParams->get('mobile_number_source');
+		$phoneField          = $notificationsParams->get('mobile_number_field');
+
+		if ($phoneSetting == 'joomla')
+		{
+			if (!empty($phoneField))
```
Comment (manojLondhe) [inline src/plugins/user/tjnotificationsmobilenumber/tjnotificationsmobilenumber.php:61]: what about else?

Comment (shindebalu) [reply src/plugins/user/tjnotificationsmobilenumber/tjnotificationsmobilenumber.php]: Done: added else.

**Pull request merged**